### DISCUSSION TITLE
Event Location/Details

### DISCUSSION THREAD

**Discussion by ddriver88** [MMRIZE/MMM-CalendarExt3#137] [Q&A] [answered]
Hi, I've got this up and running and it's great! My only wish is that I could see event details (mainly the location)

Is this possible?

**Comment by eouia** [comment, marked as answer]
Each event needs to be bigger to display all the things. And the screen is not spacious enough for that. 
There are some alternative options;
1) If your screen/mirror supports touch or mouse, you can populate the detail view by clicking the event or the day cell.
2) Or you can use other helper modules to get details. For example, MMM-CalendarExt3Agenda.
3) You can assign a colour or symbol to represent the `location` or transform the title to include the location.
